proposals/array_merge.json

```diff
@@ -0,0 +1,75 @@
+{
+    "id": "array_merge",
+    "summary": "Merge two arrays",
+    "description": "Merges two arrays into a single array by appending the second array to the first array. Array labels get discarded from both arrays before merging.",
```

Comment:
I've proposed merge to align with text_merge (without "overlap resolver") and merge_cubes (with "overlap resolver").

> wouldn't it be better then to reserve "merge" for this label-aware version?

I wondered whether we should just add this functionality here, to make it more merge_cubes like.

Reply:
FYI: back in the days when I did PHP development, its `array_merge` function was one of my pet peeves. I grew quite a disgust for it because of its schizophrenic behaviour depending on the label type (int versus strings): https://www.php.net/manual/en/function.array-merge.php 
>  If the input arrays have the same string keys, then the later value for that key will overwrite the previous one. If, however, the arrays contain numeric keys, the later value will not overwrite the original value, but will be appended. 

I would strongly recommend  to have simple concatenation and "smart" label-aware merging as separate processes. A user only needs one or the other in a given situation. It's not worth it  to unify them, it unnecessarily puts the mental overhead burden on the user.

Reply:
I'm fine with that, so I'd go with the little shorter array_concat then. Now, it hurts me that text_merge is not text_concat though (but you get a feeling why PHP is so inconsistent at times) ;-)

Reply:
Renamed from array_merge to array_concat.